Smart-contracts/.env

```
@@ -0,0 +1,16 @@
+
+HEDERA_TESTNET_PRIVATE_KEY = 0xd5652b6370ef6eedd850529def34007cbde3ddd695b150d54c2a1fb9215f6f39
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)